UPGRADE.md

```diff
@@ -256,19 +256,32 @@ See the section above on Plugins for information on plugin compatibility.
 
 Kong 1.0 introduces a new, improved migrations framework.
 It supports a no-downtime, Blue/Green migration model for upgrading
-from 0.14.x. The full migration is now split into two steps,
-which are performed via commands `kong migrations up` and
-`kong migrations finish`.
+from 0.14.x. This means that while the migration is ongoing, you will
+have two Kong clusters running, sharing the same database. The "Blue"
+cluster is running 0.14.x, the "Green" cluster is running Kong 1.0.
+
+The migrations are designed so that there is no need to fully copy
+the data, but this also means that it designed in such a way so that
```

Comment:
Faulty English on "it designed". 

Since the paragraph starts with "The migrations are designed " I think the "it" could be replaced by a "they're".